state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -3359,6 +3431,7 @@ impl<T: Config<I>, I: 'static> DepositApi<T::TargetChain> for Pallet<T, I> {
 		(ChannelId, ForeignChainAddress, <T::TargetChain as Chain>::ChainBlockNumber, Self::Amount),
 		DispatchError,
 	> {
+		// Try to get a pre-allocated channel first, if not request new one
```

Comment:
I don't think these comments are necessary, and if anything. If you do want to keep the comment, then I would put it on the `open_channel` function itself to avoid duplicating it